cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js

```diff
@@ -56,10 +56,19 @@ describe('Dump test data', () => {
 });
 
 describe('Save a gantt chart', { defaultCommandTimeout: 20000 }, () => {
-  before(() => {
-    CURRENT_TENANT.newTenant = 'global';
-  });
   beforeEach(() => {
+    if (Cypress.env('SECURITY_ENABLED')) {
```

Comment:
Making private tenant as default tenant in the beginning of this file should be good enough.